src/chrome/content/rules/JavaLand.eu.xml

```diff
@@ -0,0 +1,18 @@
+<!--
+	Timeout:
+		test.javaland.eu
+-->
+<ruleset name="JavaLand.eu">
+
+	<target host="javaland.eu" />
+	<target host="www.javaland.eu" />
+	<target host="programm.javaland.eu" />
+
+	<!-- Invalid certificate on https://javaland.eu/,
+	http://javaland.eu/ redirects to http://www.javaland.eu/ -->
```

Comment:
I get `Connection closed` / `Secure connection closed` (which is the same error but presented differently depending on your browser).
It would also be better if you could put this in the top comment for consistency.